gensim/models/wrappers/dtmmodel.py

```diff
@@ -93,7 +93,7 @@ def __init__(
             lencorpus = sum(1 for _ in corpus)
         if lencorpus == 0:
             raise ValueError("cannot compute DTM over an empty corpus")
-        if model == "fixed" and any([i == 0 for i in [len(text) for text in corpus.get_texts()]]):
+        if model == "fixed" and any([i == 0 for i in [len(text) for text in corpus]]):
```

Comment:
No need for the outer brackets `[` and `]` (generator fine).

But the whole construction feels unpythonic and strange -- simple `any(not text for text in corpus)` better?